configs/plugins/mermaid-reporter-plugin.js

```diff
@@ -84,6 +83,22 @@ const convertedSubgraphSources = (pCruiseResult) => {
   return lTree;
 };
 
+const addFocusHighlights = (pModules) => {
+  return pModules
+    .filter((pModule) =>
+      Object.prototype.hasOwnProperty.call(pModule, "matchesFocus")
+    )
+    .reduce(
+      (pAll, pModule) =>
+        (pAll += `\nstyle ${hashNodeName(pModule.source)} ${
+          pModule.matchesFocus
+            ? "fill:lime,color:black"
+            : "fill:lightgrey,color:black"
+        }`),
+      ""
+    );
+};
```

Comment:
Understanding that the color scheme is the same as that of the dot reporter, I think it would be better to add highlighting when `matchedFocus` is true and leave it as is otherwise. There are two reasons.

- mermaid.js and GitHub support dark mode. It seems better to have the background color change according to the user's environment instead of using lightgray as a uniform background color.
- The default value of mermaid.js and tools like GitHub and Notion seem to set the character limit at 50000, and it seems that you should reduce unnecessary characters as much as possible to avoid exceeding the maximum character limit.

Reply:
Agree, good point! I'll update it.